### PR TITLE
Update for insiders.

### PR DESCRIPTION
**Pull request by sean-mcmanus**

### CODE REVIEW

File: Extension/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # C/C++ for Visual Studio Code Change Log
 
-## Version 0.17.7: July 16, 2018
+## Version 0.17.7-insiders: July 12, 2018
```
Comment (bobbrow) [inline Extension/CHANGELOG.md]: don't change the changelog. it's a source of merge conflicts later.

**Review by bobbrow — CHANGES_REQUESTED** [review]

File: Extension/CHANGELOG.md (hunk 1 above)
Comment (sean-mcmanus) [reply Extension/CHANGELOG.md]: Okay.

File: Extension/package-lock.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-tE1SYtNG3I3atRVPELSGN2FJJJtPg3O/G0tycYSyzeDqdAbdLPRH089LhpWYA5M/iHeWHkVZq/b0OVKngcK0Eg==",
       "dev": true,
       "requires": {
-        "@types/node": "8.5.4"
+        "@types/node": "*"
```
Comment (bobbrow) [inline Extension/package-lock.json]: this is strange.  I wonder why this was not updated by the merge from master.

Comment (sean-mcmanus) [reply Extension/package-lock.json]: I think master is incorrect.

Comment (bobbrow) [reply Extension/package-lock.json]: no, master is correct. this is wrong.  I'll see if I can fix it

**Review by bobbrow — APPROVED** [review]

**Pull request merged**